packages/strapi-plugin-email/config/functions/bootstrap.js

```diff
@@ -30,6 +30,20 @@ module.exports = async cb => {
         return _.startsWith(node_module, 'strapi-provider-email') || _.startsWith(node_module, 'strapi-email');
       });
 
+      node_modules.filter((node_module) => {
+        return node_module.startsWith('@');
+      })
+        .forEach((orga) => {
```

Comment:
Do we really need to loop on all organization whereas we look for a specific package `@strapi/strapi-provider-email`?